content/en/tutorials/lotus-miner/cuda.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "CUDA for the Nvidia cards"
+description: "This is a step by step guide on how to configure and use CUDA architecture for Lotus instead of OpenCL for Nvidia cards."
+lead: "This is a step by step guide on how to configure and use CUDA architecture for Lotus instead of OpenCL for Nvidia cards."
+draft: false
+menu:
+    tutorials:
+        parent: "tutorials-providers"
+weight: 130
+toc: true
+---
+
+## Installing latest stable Nvidia Drivers
+
+Note that many Linux distributions provide their own packages of the NVIDIA Linux Graphics Driver in the distribution's native package management format. This may interact better with the rest of your distribution's framework, and you may want to use this rather than NVIDIA's official package.
```

Comment:
```suggestion
Many Linux distributions provide their own packages of the NVIDIA Linux Graphics Driver in the distribution's native package management format. This may interact better with the rest of your distribution's framework, and you may want to use this rather than NVIDIA's official package.
```

Don't say _Note that_.